Formula/liquidprompt.rb

```diff
@@ -10,21 +10,27 @@ class Liquidprompt < Formula
     sha256 cellar: :any_skip_relocation, all: "407ac67632d4c74525162eedfbc1033ca4ea69ca70a8ce225531d09c35d195eb"
   end
 
+  doc_url = "https://liquidprompt.readthedocs.io/en/stable/config.html"
+
   def install
-    share.install "liquidpromptrc-dist"
     share.install "liquidprompt"
   end
 
-  def caveats
+  define_method :caveats do
+    if build.head?
+      doc_url = "https://liquidprompt.readthedocs.io/en/latest/config.html"
+      head_note = " (for the HEAD version)"
```

Comment:
can you reference `#{Utils.git_head}` in here?

Reply:
I'm not sure what you mean here. I used `build.head?` because I saw that in the official documentation.